manifests/prow-test-pods/secretdefinitions/s3-credentials.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: secrets-manager.tuenti.io/v1alpha1
+kind: SecretDefinition
+metadata:
+  name: s3-credentials
+spec:
+  name: s3-credentials
+  keysMap:
+    service-account.json:
+      path: secret/data/kubernetes/pro-base/ocp4/common/3scale-pro-base-prow-s3-credentials
```

Comment:
This secret just needs the json, difference to the other one requiring secret/access key, bucket, region...?

Reply:
Yes, this secret is used by the prow jobs sidecar and only needs the json.